webpack.config.js

```diff
@@ -65,7 +65,34 @@ const webExtensionConfig = {
             // Webpack 5 no longer polyfills Node.js core modules automatically.
             // see https://webpack.js.org/configuration/resolve/#resolvefallback
             // for the list of Node.js core module polyfills.
-            assert: require.resolve('assert'),
+            assert: false,
+            buffer: require.resolve('buffer'),
```

Comment:
a lot of these might not actually be needed.  Essentially these base node dependencies arent bundled with webpack when running in the browser.  Some have replacements that work in the browser, others (like fs) don't and using them in the browser is wrong. 

xx: false means the package will exist and compile, but if used it fails at runtime.  Otherwise we have to exclude them entirely.